### PR TITLE
RS-24: New download page structure with Azure tab

### PR DESCRIPTION
**Pull request by AnthonyCvn**
- new tab for Azure
- new tab for Docker
- structure CLI and SDKs in tabs

### CODE REVIEW

**Comment by AnthonyCvn** [comment]
@atimin, please, review.

File: _pages/download/index.html [hunk 1]
```diff
@@ -4,68 +4,65 @@
 description: Download page for ReductStore and its SDKs with download links and instructions for Linux, Windows and macOS.
 permalink: /download
 ---
-<p>
-    ReductStore supports Linux, macOs and Windows on AMD64 platform.
-</p>
+
+<p>ReductStore supports Linux, macOs and Windows on AMD64 platform.</p>
```
Comment (atimin) [inline _pages/download/index.html:8]: The  information isn't correct anymore.  We're currently supporting:
* Docker: amd64, arm64, arm32
* Linux, macOs, Windows: amd64

File: _pages/download/azure.html [hunk 2]
```diff
@@ -0,0 +1,41 @@
+<h2>ReductStore Server</h2>
+<div class="hline"></div>
+
+<p>
+  ReductStore is available as an IoT module on the Azure Marketplace. This
```
Comment (atimin) [inline _pages/download/azure.html:5]: does it make sense to make it short and move the deployment instruction here -  https://docsdev.reduct.store/#azure-iot

**Comment by AnthonyCvn** [comment]
@atimin please have a look to my corrections.

**Pull request merged**